migrator.go

```diff
@@ -0,0 +1,84 @@
+package influxdb
+
+import "context"
+
+// MigrationManager is the manager of migrators.
+type MigrationManager struct {
+	Migrators []Migrator
+}
+
+// Migrate will perform the migration.
+func (mg MigrationManager) Migrate(ctx context.Context, Up bool, errCh chan<- error) {
+	for _, m := range mg.Migrators {
+		if !m.NeedMigration() {
+			return
```

Comment:
Return here would mean the first time a migrator on the list of migrators didn't need to migrate the rest of the list would never be executed.